README.md

```diff
@@ -146,13 +190,16 @@ posts for further information
     </td><td><code>Standard_A2</code></td></tr>
 
   <tr><td>kibanaCertBlob</td><td>string</td>
-    <td>A Base-64 encoded form of the certificate (.crt) to secure HTTPS communication between the browser and Kibana.</td><td><code>""</code></td></tr>
+    <td>A Base-64 encoded form of the certificate (.crt) in PEM format to secure HTTPS communication between the browser and Kibana.</td><td><code>""</code></td></tr>
```

Comment:
A PEM certificate will always be a base64 encoded DER certificate so `Base-64 encoded form` sounds superfluous I think

Reply:
I agree that PEM will already be base 64, but I'd like to keep the description here for consistency, and think it's a good idea to keep the process for including certs consistent across all the cert  parameters i.e. as a consumer of the template, if I need to add certs in whatever format, I know that I always need to base 64 encode the contents and pass that as the parameter.

I'll add an example to the README of how to pass certificates to the template, to make it clear.

Reply:
Perhaps it would be more clear if I remove the `in PEM format` part, and leave as before? This way, whether providing a PEM cert or DER cert, so long as it's base 64 encoded (_I appreciate that PEM is already base 64_ 😃), the certificate will work.